### PR TITLE
[NLU-4112] Currency resolving for EGP and SAR

### PR DESCRIPTION
**Pull request by CiaraCrowe**
Resolved the currency for Egyptian Pound EGP and Saudi Riyal SAR

### CODE REVIEW

**Comment by CiaraCrowe** [comment]
https://jenkins.ininica.com/view/Dialog%20Engine/job/NLU/job/Recognizers-Text/473/ Tests passing

**Review by Conor-Keaney — APPROVED** [review]

**Review by samhickey25 — APPROVED** [review]

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py [hunk 1]
```diff
@@ -233,7 +233,7 @@ class ArabicNumericWithUnit:
                                ("Ascension pound", "جنيه الصعود|جنيه الصعود|جنيه الصعود"),
                                ("Saint Helena pound", "جنيه سانت هيلينا|جنيه سانت هيلينا|جنيه سانت هيلينا جنيه استرليني|shp"),
                                ("Egyptian pound",
-                                "جنيه مصري|جنيه مصري|جنيه إسترليني|egp|ج.م|جنيه مصري|جنيه مصري"),
```
Comment (kevinwalshgen) [inline Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py:236]: Are you sure this was invalid: `جنيه مصري`? It does translate to 'egyptian pound', so could it be an either or scenario?

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py [hunk 2]
```diff
@@ -300,7 +300,7 @@ class ArabicNumericWithUnit:
                                 "دولارات جزر بيتكيرن|دولار جزر بيتكيرن|دولارات جزر بيتكيرن $|دولار جزيرة بيتكيرن|دولار جزيرة بيتكيرن|دولار جزيرة بيتكيرن $"),
                                ("Solomon Islands dollar",
                                 "دولار جزر سليمان|دولار جزر سليمان|جزر سليمان $|si$|sbd|دولار جزر سليمان|دولار جزيرة سليمان|جزر سليمان $"),
-                               ("Surinamese dollar", "دولار سورينام|دولار سورينامي|دولار سورينام|ريال سعودي"),
+                               ("Surinamese dollar", "دولار سورينام|دولار سورينامي|دولار سورينام|SRD"),
```
Comment (kevinwalshgen) [inline Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py]: I see in other regex patterns the currency codes are lowercase. Not sure if it matters, but should probably follow the same pattern?

File: Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py (hunk 1 above)
Comment (CiaraCrowe) [reply Python/libraries/recognizers-number-with-unit/recognizers_number_with_unit/resources/arabic_numeric_with_unit.py:236]: There was a duplicate of that word, there is one in there now

**Comment by CiaraCrowe** [comment]
Spoke with Huessien and he gave me the correct translations

**Review by kevinwalshgen — APPROVED** [review]

**Review by BobbyMartins5 — APPROVED** [review]
Nice one 👍🏾

**Comment by CiaraCrowe** [comment]
https://jenkins.ininica.com/view/Dialog%20Engine/job/NLU/job/Recognizers-Text/475/

**Pull request merged**